nativescript-angular/directives/list-view-comp.ts

```diff
@@ -160,6 +162,7 @@ export class ListViewComponent implements DoCheck, OnDestroy, AfterContentInit {
 
         if (args.view && args.view[NG_VIEW]) {
             listViewLog("onItemLoading: " + index + " - Reusing existing view");
+            console.log("onItemLoading: " + index + " - Reusing existing view");
```

Comment:
This is the code that should extract the view ref, event if the root view was "wrapped" in Stack layout